packages/react/src/ReactChildren.js

```diff
@@ -25,7 +29,7 @@ const SUBSEPARATOR = ':';
  * @param {string} key to be escaped.
  * @return {string} the escaped key.
  */
-function escape(key) {
+function escape(key: any): string {
```

Comment:
Why is key any?

Reply:
Yea, I'd type the other side, that is the broken one, as any. In this case `component: mixed` should maybe be `component: any` at the callsite.